### PR TITLE
Update message string of commands

### PR DESCRIPTION
**Pull request by LimErnest**
## Describe your changes
Add message formats for demote, promote, clear and help

## Issue ticket number and link
builds on #115 

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DemoteCommand.java [hunk 1]
```diff
@@ -25,9 +25,15 @@ public class DemoteCommand extends Command {
 
     public static final String COMMAND_WORD = "demote";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Converts an employee to a potential hire. "
-            + "Parameters: INDEX (must be a positive integer) "
-            + "\nExample: " + COMMAND_WORD + " 1";
+    public static final String MESSAGE_PURPOSE = "Converts an employee to a potential hire.";
+
+    public static final String MESSAGE_PARAMETER = "INDEX (must be a positive integer)";
```
Comment (thortol) [inline src/main/java/seedu/address/logic/commands/DemoteCommand.java]: Should this be MESSAGE_FORMAT?

File: src/main/java/seedu/address/logic/commands/DemoteCommand.java [hunk 2]
```diff
@@ -25,9 +25,15 @@ public class DemoteCommand extends Command {
...
+
+    public static final String MESSAGE_EXAMPLE = COMMAND_WORD + "1";
```
Comment (jevan13) [inline src/main/java/seedu/address/logic/commands/DemoteCommand.java]: Should there be a space before 1?

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 3]
```diff
@@ -11,8 +11,9 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
 
+    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_PURPOSE = "Deletes every entry in StaffSync";
```
Comment (InfinityTwo) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java]: Should there not be a full stop at the end?

**Review by InfinityTwo — APPROVED** [review]
Outstanding job! LGTM.

**Pull request merged**